src/DataFixtures/AppFixtures.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Account;
+use App\Entity\Category;
+use App\Entity\Comment;
+use App\Entity\Tricks;
+
+use DateTimeImmutable;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+
+class AppFixtures extends Fixture
+{
+    private UserPasswordHasherInterface $hasher;
+
+    public function __construct(UserPasswordHasherInterface $hasher)
+    {
+        $this->hasher = $hasher;
+    }
+
+    public function load(ObjectManager $manager): void
+    {
+        $user = new Account();
+        $user->setUsername('warhog76');
+
+        $user->setEmail('warhog76@free.fr');
+
+        $password = $this->hasher->hashPassword($user, 'Admin1234');
+        $user->setPassword($password);
+
+        $manager->persist($user);
+
+        $manager->flush();
+
+        $dateTime = new DateTimeImmutable('NOW');
+
+        for ($i=0; $i < 10 ; $i++)
```

Comment:
for ($i en double

Reply:
le code n'est surement pas bien placé. le but est de créer 4 catégories